storage/spanstore/utils.go

```diff
@@ -0,0 +1,17 @@
+package spanstore
+
+import (
+	"hash"
+	"math"
+)
+
+// HashBytes returns the uint64 hash value of bytes slice
+func HashBytes(h hash.Hash64, bytes []byte) uint64 {
+	h.Reset()
+	_, err := h.Write(bytes)
+	if err != nil {
+		// No downsampling when there's error hashing
```

Comment:
metric please

Reply:
Same question as logger, doesn't seem like metricsFactory is being passed down from consumer?